system_report.sh

```diff
@@ -45,6 +45,7 @@ ver_line="$(zip -v | head -n 2 | tail -n 1)";     echo "* zip: $ver_line"
 ver_line="$(tar --version | head -n 1)" ;         echo "* tar: $ver_line"
 ver_line="$(tree --version)" ;                    echo "* tree: $ver_line"
 ver_line="$(gcc --version | head -n 1)" ;         echo "* gcc: $ver_line"
+ver_line="$(convert -version | grep Version)" ;   echo "* ImageMagick: $ver_line"
```

Comment:
sorry, I requested this change because:

1. to have it unified through system reports
2. you should avoid `grep`ing the version output, as that can print more than one line - we did have issues with that in the past. When `head` can be used that's a much better solution.

Sorry for not explaining it better.